File: Estrutura_Condicional/C++/Ex1044/main.cpp
#include <iostream>
#include <algorithm>

int main() {
    int a, b;
    std::cin >> a >> b;

    int maior = std::max(a, b);
    int menor = std::min(a, b);

    if (maior % menor == 0) {
        std::cout << "Sao Multiplos" << std::endl;
    } else {
        std::cout << "Nao sao Multiplos" << std::endl;
    }

    return 0;
}

File: Estrutura_Condicional/C#/Ex1044/Main.cs
using System;

namespace MyCompiler {
    class Program {
        public static void Main(string[] args) {
            string[] valores = Console.ReadLine().Split();
            int a = int.Parse(valores[0]);
            int b = int.Parse(valores[1]);

            int maior = Math.Max(a, b);
            int menor = Math.Min(a, b);

            if (maior % menor == 0){
                Console.WriteLine("Sao Multiplos");
            } else {
                Console.WriteLine("Nao sao Multiplos");
            }
        }
    }
}